src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/AdditionalServicesOrdering.feature

```diff
@@ -0,0 +1,13 @@
+﻿Feature: Additional Services Ordering
+	As a Buyer
+    I want the Additional Services for the Catalogue Solutions in my order to be displayed in alphabetical order
```

Comment:
Similar to above, alignment of lines 2 and 3 looks out – guess it's a tabs/spaces issue.